src/Adapter/Search/SearchProductSearchProvider.php

```diff
@@ -70,10 +73,14 @@ public function runQuery(
             );
             $products = $result['result'];
             $count    = $result['total'];
+
+            Hook::exec('actionSearch', array('expr' => $queryString, 'total' => $count));
```

Comment:
Hello @kpodemski 
I'm agree with you :/ I just re-add this hook like 1.6 because it was forget.. In order to not break some modules which are plugged with this params..